### PR TITLE
✨ Adding vidoomy network to amp-ad 

### PR DESCRIPTION
**Pull request by mortas-vidoomy**
Added vidoomy (https://vidoomy.com) network to amp-ads

### CODE REVIEW

**Comment by mortas-vidoomy** [comment]
@googlebot I signed it!

File: ads/vidoomy.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
Comment (powerivq) [inline ads/vidoomy.js]: It's officially 2020!

File: ads/vidoomy.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            widthMbl: data['widthMbl'] || 300,
+            heightMbl: data['heighMbl'] || 200,
+            appearAt: data['appearAt'] || 'right',
+            zoneId: data['zoneId'],
+            zoneIdMbl: data['zoneIdMbl']
+        }, '', data['uniqueId'], 5000, global, 'imasdk');
+    }
+
+  });
+}
```
Comment (powerivq) [inline ads/vidoomy.js]: nit: newline

File: 3p/integration.js [hunk 3]
```diff
@@ -579,6 +580,8 @@ register('zergnet', zergnet);
 register('zucks', zucks);
 register('speakol', speakol);
 
+register('vidoomy',vidoomy);
```
Comment (powerivq) [inline 3p/integration.js]: nit: space between arguments

File: ads/vidoomy.js (hunk 1 above)
Comment (mortas-vidoomy) [reply ads/vidoomy.js]: My bad, changed!

File: ads/vidoomy.js (hunk 2 above)
Comment (mortas-vidoomy) [reply ads/vidoomy.js]: Added newline

File: 3p/integration.js (hunk 3 above)
Comment (mortas-vidoomy) [reply 3p/integration.js]: Added space, and also placed on alphabetically order

**Comment by mrjoro** [comment]
Is this PR still active?

/cc @ampproject/wg-ads

**Comment by powerivq** [comment]
Closing inactive PRs. Please reopen if still wanting to merge it.

**Pull request closed without merging**